package-lock.json

```diff
@@ -723,8 +731,7 @@
     "@types/node": {
       "version": "12.12.21",
       "resolved": "https://registry.npmjs.org/@types/node/-/node-12.12.21.tgz",
-      "integrity": "sha512-8sRGhbpU+ck1n0PGAUgVrWrWdjSW2aqNeyC15W88GRsMpSwzv6RJGlLhE7s2RhVSOdyDmxbqlWSeThq4/7xqlA==",
-      "dev": true
```

Comment:
Not ideal. `@rdfjs/data-model` has a hard dependency on `@types/rdf-js` as it include its types. If it were a peer dependency, would that work? (`@types/node` is a local dev dependency.)

Reply:
I guess peer dependencies would help making this dependency optional; but is it truly optional if the types are actually used? The intent of peer dependencies seems to me to specify compatibility without implying code inclusion.